vision/style_transfer/fast_neural_style/conversion.ipynb

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
this notebook should be linked from the readme, or alternatively combined with the existing notebook